hipify-clang/CMakeLists.txt

```
@@ -111,6 +111,12 @@ if (HIPIFY_CLANG_TESTS)
         @ONLY
     )
 
+    configure_file(
+        ${CMAKE_CURRENT_LIST_DIR}/../tests/hipify-clang/lit.cfg.in
+        ${CMAKE_CURRENT_LIST_DIR}/../tests/hipify-clang/lit.cfg
```

```suggestion
        ${CMAKE_CURRENT_BINARY_DIR}/../tests/hipify-clang/lit.cfg
```
you should not create parsed in source directory tree

---

Agree. The request is updated. I've just moved load_config call for lit.site.cfg a bit earlier in lit.cfg to get initialized cuda_version and cuda_version_major. So there is no any need in lit.cfg.in (removed).